tests/helix.targets

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+      <PackageTestRuntimeFolder>$(PackagesDir)test.Microsoft.NETCore.Runtime.CoreClr\1.1.1</PackageTestRuntimeFolder>
+      <BUILDTOOLS_OVERRIDE_RUNTIME>$(PackageTestRuntimeFolder)</BUILDTOOLS_OVERRIDE_RUNTIME>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <CoreRootFiles Include="$(CORE_ROOT)\*.*" />
+  </ItemGroup>
+
+  <Target Name="ReplaceProjectLockJson"
+          BeforeTargets="CopyTestToTestDirectory">
+    <PropertyGroup>
+      <ProjectLockJson></ProjectLockJson>
+    </PropertyGroup>
+    <ItemGroup>
+      <TestNugetProjectLockFile Include="$(SourceDir)$(Category)\**\project.lock.json"/>
+    </ItemGroup>
+  </Target>
+
+
+  <Target Name="CopyProductInPackagesFolder"
+          BeforeTargets="CopyTestToTestDirectory">
+
+    <Copy
+      SourceFiles="@(runtimefiles)"
+      DestinationFolder="$(PackageTestRuntimeFolder)"
+      SkipUnchangedFiles="false"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)">
+    </Copy>
+ 
+  </Target>
+
+  <Target Name="AddFilesToAssemblyList" 
+          BeforeTargets="CreateAssemblyListTxt">
+
+    <ItemGroup>
+
+      <RuntimeFiles Include="$(PackageTestRuntimeFolder)\*" />
+      <XunitConsoleExe Include="$(PackagesDir)xunit.runner.console\**\xunit.console.exe" />
+
+      <_TestCopyLocalByFileNameWithoutDuplicates Include="@(RuntimeFiles)">
+        <NugetPackageId>test.Microsoft.NETCore.Runtime.CoreClr</NugetPackageId>
+        <SourcePath>%(RuntimeFiles.Identity)</SourcePath>
+      </_TestCopyLocalByFileNameWithoutDuplicates>
+
+      <_TestCopyLocalByFileNameWithoutDuplicates Include="@(XunitConsoleExe)">
+        <NugetPackageId>xunit.runner.console</NugetPackageId>
```

Comment:
You should add a TODO here and file an issue to track how the test runs should be done across all OS platforms per our discussion.